src/table/utils/handle-key-press.js

```diff
@@ -164,6 +165,7 @@ export const bodyHandleKeyPress = ({
   switch (evt.key) {
     case 'ArrowUp':
     case 'ArrowDown': {
+      evt.key === 'ArrowUp' && handleNavigateTop([cell.rawRowIdx, cell.rawColIdx], rootElement);
```

Comment:
Add or extend a test to see if `handleNavigateTop` has been called

Reply:
Good point. Gonna do

Reply:
Done